src/main/java/org/broadinstitute/hellbender/metrics/InsertSizeMetricsArgumentCollection.java

```diff
@@ -16,24 +16,36 @@
 // TODO: user argument validation (eg. maxMADTolerance)
 
 /**
- * MetricsArgumentCollection argument collection for InsertSize metrics. All members should be
- * instantiable as command line arguments.
+ * ArgumentCollection for InsertSizeMetrics collectors.
  */
-public class InsertSizeMetricsArgumentCollection extends MetricsArgumentCollection {
-    @Argument(doc = "A local path to PDF file where histogram plot will be saved in.",
-            shortName = "HIST",
-            fullName = "HistogramPlotPDF",
-            optional = false)
-    public String histogramPlotFile = null;
+public class InsertSizeMetricsArgumentCollection extends MetricsArgumentCollection implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Argument(fullName = "histogramPlotFile",
+            shortName="H",
+            doc="File to write insert size Histogram chart to.")
+    public String histogramPlotFile;
 
-    @Argument(doc = "Generate mean, sd and plots by trimming the data down to MEDIAN + maxMADTolerance*MEDIAN_ABSOLUTE_DEVIATION. " +
+    @Argument(doc="Generate mean, sd and plots by trimming the data down to MEDIAN + DEVIATIONS*MEDIAN_ABSOLUTE_DEVIATION. " +
```

Comment:
Also a result of consolidating two similar but not identical implementations. I'll revert this to the previous doc string so it matches the argument name.